jobs/eventsPusher.ts

```diff
@@ -17,6 +17,7 @@ export default class EventsPusher {
   getCredentials = (serviceAccountKey: string): unknown => {
     try {
       return JSON.parse(serviceAccountKey)
+      // eslint-disable-next-line  @typescript-eslint/no-unused-vars
     } catch (e) {
```

Comment:
```suggestion
    } catch (_e) {
```

Reply:
$npm run lint -> /GITHUB/WPIP/hmpps-welcome-people-into-prison-ui/jobs/eventsPusher.ts
  20:14  warning  '_e' is defined but never used  @typescript-eslint/no-unused-vars